profiles/common/modules/features/multisite_maxlength/README.md

```diff
@@ -0,0 +1,59 @@
+Multisite Maxlength Feature
+======================
+
+Multisite Maxlength is a feature that provides to fields of a content type the
+functionality of limiting and validating their maximum length in the edit form,
+before submission, and showing a counter displaying number of characters left
+before the maximum length is reached.
+
+# Installation
+
+The feature can be activated using feature set menu
+(/admin/structure/feature-set).
+
+As it is needed to configure a field's settings in a content-type to enable
+this functionality, which is usually reserved to the administrator role, only
+users with the administrator role can activate this functionality to show a
+counter with characters left to reach the field maximum length.
+
+# Usage
+
+## Field types
+
+Widgets currently accepted are:
+- Text area (multiple rows)
+- Text area with a summary
+- Text field
+
+## Activation
+Administrators can activate this functionality at the field settings edition
+page, which can be reached from the ¡§manage fields¡¨ tab of the content type
+settings form. There are two different set ups for this functionality depending
+on the widget type:
+
+### Text field
+This is the easiest option. You only have to tick the checkbox in order to
+activate the counter, which will use the maximum length value of the field.
+It is possible to customize the message that will be shown with the counter.
+
+### Text area and Text area with a summary
+For these widgets there are more settings options. In this case, you have to
+enter a value into the 'Maximum length' setting to activate the counter.
+If this field is left empty, the counter won't be activated.
+
+Additional settings:
+
+- Force text truncate:
+  Check this option if you want that the html (or the text) that the user
+  inserts into the field to be truncated.
```

Comment:
This should become either:
if you want the html ... to be truncated
or
if you want that the html ... will be truncated